### PR TITLE
Upvote buttons mobile

### PR DESCRIPTION
**Pull request by faebser**
This is the pull request for #642. I changed both the voting buttons on the link as the ones for the comments. They are both slightly bigger and easier to hit on mobile.

Please have a look and let me know if I any more changes are needed.

### CODE REVIEW

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -389,11 +389,13 @@ var Lobsters = new _Lobsters();
 
 $(document).ready(function() {
   var $olcomments = $("ol.comments");
-  $olcomments.on("click", ".comment a.downvoter", function() {
+  $olcomments.on("click", ".comment .voters > a + div + a", function() {
```
Comment (arp242) [inline app/assets/javascripts/application.js.erb]: These kind of selectors seem very fragile to me. Actually, looking at the HTML I'm not even sure if I fully understand what this is matching exactly. Is it not possible to keep the simple `a.downvoter` match?

File: app/assets/javascripts/application.js.erb [hunk 2]
```diff
@@ -389,11 +389,13 @@ var Lobsters = new _Lobsters();
...
+    console.log("downvote");
```
Comment (arp242) [inline app/assets/javascripts/application.js.erb]: Looks like you left this in by accident, as well as another `console.log("upvote");` a bit further down.

Comment (faebser) [reply app/assets/javascripts/application.js.erb]: silly me, will get removed asap

File: app/assets/javascripts/application.js.erb (hunk 1 above)
Comment (faebser) [reply app/assets/javascripts/application.js.erb]: I had to add an additional container to the page to get the desired result of having a bigger tap area and out of a personal preference I did not assign a class to this container. The `.comment .voters > a + div + a` just matches on an `a` that is located after an `a` and a `div` on the first level of the voters `div`. I do not expect that the voters container will see much changes in the future. But you not immediately groking the selector shows that it is too complicated so I'm gonna add a class to the outer container and clean up the selector.

**Comment by faebser** [comment]
Is there something stopping this request from being merged?

**Comment by pushcx** [comment]
This adds a block-level `div` to the inline-level `a`, so it's invalid markup... but then you change `a` to be laid out as a block, so why add the div at all?

**Comment by faebser** [comment]
You are right, that is invalid markup. I'm going to fix that.

The ```div``` contains the graphics (the arrow) while the surrounding ```a``` actually catches the click. I wanted to keep the ```a``` for accessibility reasons.

**Comment by pushcx** [comment]
Ping @faebser - anything I can help with?

**Comment by faebser** [comment]
Sorry, my life is taking up most of my time right now. I will try to do this next week.

**Comment by pushcx** [comment]
Our CSS and JS have been substantially rewritten since this PR was last active, so I'm going to close it out. But anyone is welcome to pick it up and reopen this or a new PR, the inconvenience still exists.

**Pull request closed without merging**